go.mod

```
@@ -1,3 +1,3 @@
 module github.com/loong/go-concurrency-exercises
 
-go 1.19
+go 1.22
```

Ще краще змінити на останню версію:
```suggestion
go 1.23.0
```